package.json

```diff
@@ -11,7 +11,7 @@
     "deploy": "npm run build && gh-pages --dist build/",
     "live": "elm-live src/Main.elm -p 3000 -d public/ -H -- --output=public/app.js",
     "optimize": "uglifyjs build/app.js -c $npm_package_uglify_params | uglifyjs -m --output=build/app.js",
-    "sass": "sass src/Main.scss:public/main.css --source-map-urls=absolute --watch",
+    "sass": "sass src/Main.scss:public/main.css --source-map-urls=absolute --watch --poll",
```

Comment:
What is poll ?

Reply:
i have cache problems with sass, and this argument fix (a little) :(

Reply:
Ok :/